utils/author_catalog.py

```diff
@@ -64,6 +77,29 @@ def create_catalog(filepath: str = catalog_file_path) -> Dict:
     for row in read_csv(filepath):
         book = parse_data_row(row)
         if is_selected_author(book):
+            if log:
+                create_log(book)
+            if ";" in book.author:
+                book.author = book.author.split(";")[0]
+                # if author_book_catalog[book.author]:
+                #     for entry in author_book_catalog[book.author]:
+                #         if not book.title == entry.title:
+                #             author_book_catalog[book.author].append(book)
```

Comment:
unless it is really necessary, try not to check out commented code. remove it if possible.